src/routes/plans/[id]/+page.svelte

```diff
@@ -413,6 +472,14 @@
       <PlanMergeRequestsStatusButton user={data.user} />
     </svelte:fragment>
     <svelte:fragment slot="right">
+      <ActivityStatusMenu
```

Comment:
Now that we have like 7 things in this navbar i'm wondering if we should get rid of that 10px gap between all of the items to make more room for the plan title and merge request stuff? Looks like the gap is gone now in the mockups too.